src/scripts/shop/Shop.ts

```diff
@@ -4,8 +4,14 @@
 class Shop {
     items: KnockoutObservableArray<Item>;
```

Comment:
Why an observable array? As far as I know, the items in a shop will always be the same, so I don't see what benefit we get from making the array observable.